skabase/SKATestDevice/SKATestDevice/SKATestDevice.py

```diff
@@ -179,6 +213,78 @@ def RunGroupCommand(self, argin):
         return argout
         # PROTECTED REGION END #    //  SKATestDevice.RunGroupCommand
 
+    def devLogMsg(self, devlogmsg, devloglevel):
+
+        if self._element_logging_level >= 1 and devloglevel == 1:
+            self.fatal_stream("%s", devlogmsg)
+        elif self._element_logging_level >= 2 and devloglevel == 2:
+            self.error_stream("%s", devlogmsg)
+        elif self._element_logging_level >= 3 and devloglevel == 3:
+            self.warn_stream("%s", devlogmsg)
+        elif self._element_logging_level >= 4 and devloglevel == 4:
+            self.info_stream("%s", devlogmsg)
+        elif self._element_logging_level >= 5 and devloglevel == 5:
+            self.debug_stream("%s", devlogmsg)
+    #     else:
+    #         pass
+    #
+        if self._central_logging_level >= 1 and devloglevel == 1:
+            self.fatal_stream("%s", devlogmsg)
```

Comment:
If both central and element logger need to receive this message, we will have called `self.fatal_stream` twice, so both will get the message twice.  This is part of the problem due to the missing core TANGO functionality for different levels per target.

In the interim, doing a single log call would be better.  Something like:
```python
min_tls_logging_level = min(self._central_logging_level, self._element_logging_level)
if  min_tls_logging_level >= 1 and devloglevel == 1:
    self.fatal_stream("%s", devlogmsg)
elif  min_tls_logging_level >= 2 and devloglevel == 2:
    ...
```

Reply:
Yes, Anton I was actually facing this issue of getting same log twice.
But in the suggested approach, if self._central_logging_level is 3 and self._element_logging_level is 4 then we get logs only upto level 3. Should I proceed with this?

Reply:
Could go with `max` instead of `min` then.  Either way the solution is imperfect.  Either a Logger get fewer logs than they want, or they get more.